app/src/main/java/nu/yona/app/ui/frinends/TimelineStickyAdapter.java

```diff
@@ -125,6 +109,40 @@ public void onBindViewHolder(final TimelineHolder holder, int position)
 		}
 	}
 
+
+	private void setUpTimeFrameControlHolderView(DayActivity dayActivity, TimelineHolder holder)
+	{
+		if (dayActivity.getTimeZoneSpread() != null)
+		{
+			holder.getmTimeFrameGraph().chartValuePre(dayActivity.getTimeZoneSpread());
+		}
+		updateProfileImage(holder, dayActivity);
+	}
+
+	private void setUpTimeBucketControlHolderView(DayActivity dayActivity, TimelineHolder holder)
+	{
+		int maxDurationAllow = (int) dayActivity.getYonaGoal().getMaxDurationMinutes();
+		if (maxDurationAllow > 0)
```

Comment:
Any idea what this check is for? The max duration can never be negative. In case of a no-go goal, it is zero, but that you already classify as a no-go goal, so that won't come here.

Reply:
Removed the check.